DataFormats/Provenance/interface/BranchDescription.h

```diff
@@ -158,9 +158,11 @@ namespace edm {
       BranchID switchAliasForBranchID_;
 
       // A TypeWithDict object for the wrapped object
+      // This is set if and only if the dropped_ is false
```

Comment:
Would we do away with the use of TypeWithDict here for a later PR?

Reply:
I would do that in a later PR.